features/bevragen/autorisatie-gba.feature

```diff
@@ -645,16 +636,61 @@ Functionaliteit: autorisatie voor het gebruik van de API
+
+    @fout-case
+    Scenario: Autorisatie voor ad hoc gegevensverstrekking gaat morgen in
+      Gegeven de afnemer met indicatie '000008' heeft de volgende 'autorisatie' gegevens
+      | Rubrieknummer ad hoc (35.95.60) | Medium ad hoc (35.95.67) | Datum ingang (35.99.98) |
+      | 010120 010210                   | N                        | morgen                  |
+      En de geauthenticeerde consumer heeft de volgende 'claim' gegevens
+      | naam         | waarde |
+      | afnemerID    | 000008 |
+      En de persoon met burgerservicenummer '000000024' heeft de volgende gegevens
+      | geboortedatum (03.10) | geslachtsnaam (02.40) | voornamen (02.10) | geslachtsaanduiding (04.10) |
+      | 19830526              | Maassen               | Pieter            | M                           |
+      Als gba personen wordt gezocht met de volgende parameters
+      | naam                | waarde                          |
+      | type                | RaadpleegMetBurgerservicenummer |
+      | burgerservicenummer | 000000024                       |
+      | fields              | burgerservicenummer             |
+      Dan heeft de response een object met de volgende gegevens
+      | naam     | waarde                                                      |
+      | type     | https://datatracker.ietf.org/doc/html/rfc7231#section-6.5.3 |
+      | title    | U bent niet geautoriseerd voor het gebruik van deze API.    |
+      | status   | 403                                                         |
+      | detail   | Niet geautoriseerd voor ad hoc bevragingen.                 |
+      | code     | unauthorized                                                |
+      | instance | /haalcentraal/api/brp/personen                              |
+
+    Scenario: Autorisatie voor ad hoc gegevensverstrekking is vandaag ingegaan
+      Gegeven de afnemer met indicatie '000008' heeft de volgende 'autorisatie' gegevens
+      | Rubrieknummer ad hoc (35.95.60) | Medium ad hoc (35.95.67) | Datum ingang (35.99.98) |
+      | 010120 010210                   | N                        | vandaag                 |
+      En de geauthenticeerde consumer heeft de volgende 'claim' gegevens
+      | naam         | waarde |
+      | afnemerID    | 000008 |
+      En de persoon met burgerservicenummer '000000024' heeft de volgende gegevens
+      | geboortedatum (03.10) | geslachtsnaam (02.40) | voornamen (02.10) | geslachtsaanduiding (04.10) |
+      | 19830526              | Maassen               | Pieter            | M                           |
+      Als gba personen wordt gezocht met de volgende parameters
+      | naam                | waarde                          |
+      | type                | RaadpleegMetBurgerservicenummer |
+      | burgerservicenummer | 000000024                       |
+      | fields              | burgerservicenummer             |
+      Dan heeft de response een persoon met de volgende gegevens
```

Comment:
Algemene vraag: is het niet handiger om bij autorisatie scenario's waar de afnemer is geautoriseerd om alleen te valideren dat de response een persoon heeft in plaats van de gevraagde fields? Dat de fields wordt geleverd wordt al gevalideerd in andere scenario's

Reply:
hoe ziet de Dan stap er dan uit?

Reply:
Dan heeft de response '1' persoon. Zie: https://github.com/BRP-API/Haal-Centraal-BRP-bevragen/blob/develop/features/docs/dan-stap-definities.feature regel 513